src/ECAA80.c

```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
 INCLUDE_ASM(s32, "ECAA80", func_8024199C_ECB69C);
 
 // ECAA80_ItemChoice_SaveSelected (needs data?)
-INCLUDE_ASM(s32, "ECAA80", func_802419F0_ECB6F0);
+
+extern s32 ECAA80_ItemChoice_HasSelectedItem;
+extern s32 ECAA80_ItemChoice_SelectedItemID;
+
+s32 func_802419F0_ECB6F0(Evt *arg0) {
+    ECAA80_ItemChoice_SelectedItemID = dead_evt_get_variable(arg0, *arg0->ptrReadPos);
```

Comment:
```suggestion
    ECAA80_ItemChoice_SelectedItemID = dead_evt_get_variable(arg0, *arg0->ptrReadPos++);
```

Reply:
you need to use an args temp and increment that instead (search for other functions that return ApiStatus_DONE2) - this applies for all funcs in this PR

Reply:
Ok, makes sense. Those changes are now implemented.